castervoice/lib/ctrl/mgr/rule_formatter.py

```diff
@@ -1,10 +1,14 @@
 import re
 
-def set_rdescript(mapping, rcn):
+def _set_rdescripts(mapping, rcn):
     for spec, action in mapping.items():
         # pylint: disable=no-member
-        if hasattr(action, "rdescript") and action.rdescript is None:
-            mapping[spec].rdescript = _create_rdescript(spec, rcn)
+        _set_rdescript(action, spec, rcn)
+
+
+def _set_rdescript(action, spec, rcn):
```

Comment:
A small suggestion `_set_rdescript` might be better renamed to `_set_the_rdescript` That helps differentiate function names.

Reply:
good call i've committed this